core/tests/test_models.py

```diff
@@ -0,0 +1,198 @@
+import secrets
+import string
+from collections.abc import Generator
+
+import pytest
+from django.contrib.auth import get_user_model
+
+from core.models import Community, Post, PostVote, Tag
+
+User = get_user_model()
+
+UPVOTE_SCORE = 1
+DOWNVOTE_SCORE = -1
+MULTIPLE_VOTES_SCORE = 2
+SCORE_3 = 3
+MIXED_VOTES_SCORE = 0

```

Comment:
Please change those static names to be more meaningful.....
Ie SCORE_3 says what is score number. 
MIXED_VOTES_SCORE requires lookup so the test readability is bad.
Actually, are those needed? Seems it will be more readable to simply use numbers where needed, don't you think?